tests/foreman/ui/test_docker.py

```diff
@@ -384,11 +694,33 @@ def test_promote_docker_repo_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        repo_name = gen_string('utf8')
+        lce = entities.LifecycleEnvironment(
+            organization=self.organization).create()
+        with Session(self.browser) as session:
+            content_view = entities.ContentView(
```

Comment:
I'd rather move all the API creations ouside of the browser's session - it's more logical and adds readability. But that's definitely a nitpick.